ndoh_hub/test_utils.py

```diff
@@ -21,3 +27,52 @@ def test_msisdn_to_whatsapp_id(self):
         self.assertEqual(msisdn_to_whatsapp_id("082 000 1001"), "27820001001")
         self.assertEqual(msisdn_to_whatsapp_id("27820001001"), "27820001001")
         self.assertEqual(msisdn_to_whatsapp_id("+27820001001"), "27820001001")
+
+
+class TestSendWhatsappTemplateMessageTrue(TestCase):
+    @responses.activate
+    def test_send_whatsapp_template_message_number_on_whatsapp(self):
+        """
+        Send a template to Whatsapp
+        """
+        parameters = {"type": "text", "text": "test template send"}
+        namespace = "test"
+        msisdn = "+27820001001"
+        template_name = "test template"
+
+        responses.add(
+            method=responses.POST,
+            url="http://turn/v1/messages",
+            json={"messages": [{"id": "gBEGkYiEB1VXAglK1ZEqA1YKPrU"}]},
+            status=200,
+        )
+
+        response = send_whatsapp_template_message(
+            msisdn, namespace, template_name, parameters
+        )
+
+        self.assertEqual(response, "Whatsapp")
+
+
+class TestSendWhatsappTemplateMessageFalse(TestCase):
```

Comment:
This could be in the same class as above and maybe be renamed to `TestSendWhatsappTemplateMessage`